packages/rocketchat-livechat/app/client/lib/chatMessages.coffee

```diff
@@ -76,20 +76,32 @@ class @ChatMessages
 			rid ?= visitor.getRoom(true)
 
 			sendMessage = (callback) ->
-				msgObject = { _id: Random.id(), rid: rid, msg: msg, token: visitor.getToken() }
+				msgObject = {
+					_id: Random.id(),
+					rid: rid,
+					msg: msg,
```

Comment:
Just one tip, do you know we can use [property shorthand](http://es6-features.org/#PropertyShorthand)?
Example:
```javascript
msgObject = {
  _id: Random.id(),
  rid,
  msg,
  ...
}
```